packages/cita-signer/package.json

```diff
@@ -16,9 +16,11 @@
     "test:v2": "cross-env VERSION=2 ../../node_modules/.bin/jest"
   },
   "dependencies": {
+    "ec": "^0.0.1",
     "elliptic": "^6.4.0",
     "fstream": "1.0.12",
     "google-protobuf": "^3.5.0",
+    "sm-series-crypto": "^0.2.0",
```

Comment:
@kaikai1024 Please evaluate the reliability of this package

Reply:
I'm not sure. The license is ok.